packages/rpc-subscriptions-api/src/__tests__/vote-notifications-test.ts

```diff
@@ -9,8 +9,13 @@ describe('voteNotifications', () => {
         rpc = createLocalhostSolanaRpcSubscriptions();
     });
 
-    it('produces vote notifications', async () => {
-        expect.assertions(1);
+    // FIXME: Re-enable once https://github.com/anza-xyz/agave/pull/6645 is live.
```

Comment:
Backports presently requested in:

* https://github.com/anza-xyz/agave/pull/6680
* https://github.com/anza-xyz/agave/pull/6681